### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by echo66** [motools/musicontology.com#4]
Greetings!

Would it be possible to update the live website with the documentation that is missing? For example, individuals like "Single" have a description in the n3 file but not in the website.

**Comment by moustaki** [comment]
Good point - it looks like some individuals are missing from the main documentation. I'll investigate. Thanks!
